Directory.Packages.props

```diff
@@ -28,7 +28,7 @@
     <PackageVersion Include="MSTest" Version="3.10.4" />
     <PackageVersion Include="GitVersion.MSBuild" Version="6.4.0" />
     <PackageVersion Include="Duende.IdentityModel" Version="7.1.0" />
-    <PackageVersion Include="DnsClient" Version="1.8.0" />
```

Comment:
Is there a reason you need to downgrade this?

Reply:
oh, that's a merge oopsie my bad, but yes. the updated dependency causes issues on .NET Standard 2.0 targets (UWP primarily) because it pulls in .NET 5 packages in the Microsoft.Win32 namespace, will remove this change for this PR though

Reply:
Just another reason why I hate netstandard2.0, it means nothing at this point, lol.

I rather not pull in more dependencies anyway, it would have been better to make it an interface that could implement that library. I'm fine downgrading the library for now in its own PR if you like, and whenever I get time I can try to remove that outright and make it its own library, since it's not that much code and you can implement it however you want for your apps.